yt/frontends/gamer/io.py

```diff
@@ -91,13 +92,39 @@ def _read_particle_fields(self, chunks, ptf, selector):
                     data = self._group_particle[field][start:end]
                     yield (ptype, field), data[mask]
 
+    @lru_cache(maxsize=8)  # noqa
```

Comment:
why the noqa comment ? (making it specific to the warning(s) you want to filter, e.g. `noqa: E555`, would answer my question)